src/realm/CMakeLists.txt

```diff
@@ -319,6 +319,9 @@ set_target_properties(Storage PROPERTIES
 )
 
 target_compile_options(Storage PUBLIC ${REALM_SANITIZER_FLAGS})
```

Comment:
It seems weird that we are doing this unconditionally, but then doing it again on line 331 inside an `if(NOT MSVC)` block. But I didn't want to change that as part of this commit. 🤷